openid-4-verifiable-presentations-1_0.md

```diff
@@ -280,16 +280,18 @@ One exception to this rule is `transaction_data` parameter, and the wallets that
 This specification defines the following new request parameters:
 
 `presentation_definition`:
-: A string containing a Presentation Definition JSON object. See (#request_presentation_definition) for more details.
+: A JSON object containing a Presentation Definition. See (#request_presentation_definition) for more details.
```

Comment:
I don't think this change is helpful. This should clarify that the encoding varies between the query string (where it is a JSON-encoded string) and the request object and DC API call (where it is an object and it is not additionally JSON-encoded).

Reply:
I sort of understand this perspective @danielfett, however I'm struggling to see what other interpretation a developer might have about how to convey a JSON object in a query parameter value?

Reply:
@danielfett Are we at least agreed the current text is bad? It has resulted in people doing this in request objects:

`"dcql_query": "{\"id\":\"ffc717a3-abaf-4ec3-9c55-a9b8e998874c\",\"name\":\"A`

The way I've done it here (combined with the text below explaining how to put JSON objects into the url query) seems consistent with how it's done in RAR ( https://datatracker.ietf.org/doc/html/rfc9396#section-2 ), and we have the example to try and help people struggling to interpret it too.

Reply:
When this is a DC API call, this is an object (not JSON-encoded).
```suggestion
: An object containing a Presentation Definition. See (#request_presentation_definition) for more details.
```

Reply:
If we're going to make that change for that reason I suggest doing it consistently everywhere we use the term 'JSON object' in a separate PR.